reviews/npm/iter-server/review-1.md

```diff
@@ -0,0 +1,54 @@
+<!--
```

Comment:
This is great! Thanks Dilan! We changed the metadata format slightly, would you mind updating, should be below. The "Associated-With-Project" field means that you don't have an inherent bias for/against the project, and "Compensation-Source" means that you weren't paid/compensated to do this review.
```
---
Publication-State: Draft
Access: Public
Reviewers:
- Name: Dilan Bhalla
  Associated-With-Project: false
  Compensation-Source: none
Domain: Security
Methodology:
- Web-Search
- Code-Review
- External
Issues-Identified: Severe
Package-URLs:
- pkg:npm/iter-server@1.0.0
Review-Date: 2021-02-12
Scope: Implementation/Full
Schema-Version: 1.0
SPDX-License-Identifier: CC-BY-4.0
---
```